### DISCUSSION TITLE
Why is Actions in .github folder not included in ./**/*.yaml

### DISCUSSION THREAD

**Discussion by andreas-lundgren-qlucore** [prettier/prettier#15316] [Q&A] [answered]
When doing `npx prettier ./**/*.yaml --check`, my github action workflows, as example `./github/workflows/run-unit-tests.yaml`, are not included in the analysis.

If I explicitly calling `npx prettier ./.github/**/*.yaml --check`, they are included.

I have an empty .prettierignore (and .gitignore) file.

Is there a way I can obtain a list over all files that was analyzed?
Is there a specific syntax to include dot folders or dot directories) when running prettier?

**Comment by kachkaev** [comment, marked as answer]
👋 @andreas-lundgren-qlucore! This might be to do with glob unwrapping in bash. Try running 

```bash
npx prettier "./**/*.yaml" --check
## or
npx prettier './**/*.yaml' --check
## instead of
npx prettier ./**/*.yaml --check
```

I don’t know bash well, but I guess that in the first two cases Prettier gets a glob and in the latter case it gets a list of files provided by bash.

**Reply by andreas-lundgren-qlucore** [reply]
Thank you so much! You were spot on. Let me know if I can do anything back for the community!
